wireless_modules/middle_module/config.example.py

```diff
@@ -0,0 +1,9 @@
+from machine import Pin
+
+ESSID = "<WIFI NETWORK NAME HERE>"
+PASSWORD = "<WIFI PASSWORD HERE>"
+
+MQTT_BROKER = "<Broker domain address or IP address>"
+
+builtin_LED = Pin(2, Pin.OUT)
+
```

Comment:
Looks good! Like before I would suggest moving this to the middle_module folder so it can be used for all the modules.

Reply:
I assume you meant the `wireless_modules` folder :joy: 
Now I think about it, `mqtt_client.py` can be moved too. The only thing specific to the middle module is `main.py`!

Reply:
hahaha true, I agree with all this, it shall be moved :)